R/use_tic.R

```diff
@@ -307,7 +307,7 @@ use_tic <- function(wizard = interactive(),
 
   rule(left = "tic")
 
-  use_tic_r(repo_type = detect_repo_type())
+  use_tic_r(repo_type = detect_repo_type(), deploy_on = )
```

Comment:
Is this intended?

Reply:
Thanks, wondering how that even passed CI. Let's default to "none".

Reply:
Ah its in `use_tic()` which has no tests. We really need some..